mkdocs.yml

```diff
@@ -117,6 +117,7 @@ nav:
       - Uninstall: "admin/uninstall.md"
       - Release Notes:
           - "admin/release_notes/index.md"
+          - v1.8.1: "admin/release_notes/version_1.8.1.md"
```

Comment:
Patch releases don't get there own file, can you just prepend the contents into the 1.8 one

Reply:
Fixed, thanks